lib/ember_cli/build_monitor.rb

```diff
@@ -32,7 +32,7 @@ def wait!
     attr_reader :name, :paths
 
     def complete?
-      !paths.lockfile.exist?
+      !paths.lockfile.exist? && paths.dist.join("index.html").exist?
```

Comment:
@patcoll the lockfile is removed by https://github.com/rondale-sc/ember-cli-rails-addon/blob/cadb98ad22462664fda501fb7213649123fd4c63/index.js#L63-L65.

I agree, the build should block until `index.html` exists.